<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_PDU_worker</class>
 <widget class="QWidget" name="widget_PDU_worker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_fw">
     <property name="text">
      <string>fw</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_log"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_radio_in">
     <property name="text">
      <string>EF06</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_electro_spec">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_radio_out">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
